src/utils/coordinates.js

```diff
@@ -48,7 +48,7 @@ module.exports.stringify = stringify;
  * @returns {bool}
  */
 module.exports.isCoordinate = function (value) {
-  return regex.test(value);
+  return value[0] !== '#' && regex.test(value);
```

Comment:
So coordinates will always have whitespaces in between the numbers. Colors won't. I see that the whitespace characters uses star (`*`) rather than (`+`) which will match even 0 whitespace characters. Need to change those to `+` to force at least 1 whitespace character.

try `/\s*(-?\d*\.{0,1}\d+)\s+(-?\d*\.{0,1}\d+)\s+(-?\d*\.{0,1}\d+)\s*/;`